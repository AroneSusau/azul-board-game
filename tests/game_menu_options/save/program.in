1
a
b
save saves/game_menu_options:save.azul
