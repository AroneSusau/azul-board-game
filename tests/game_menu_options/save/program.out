
      db    8888P 8    8 8    
    dPYb     dP  8    8 8    
  dPwwYb   dP   8b..d8 8    
dP    Yb d8888 `Y88P' 8888

Info: No seed argument detect, using seed 0 instead.
Enter a number
1. Start Game
2. Load Game
3. Credits
4. Exit
> Enter a name for player 1
> Enter a name for player 2
> 
Let us Play!

=== Help Menu ===
command: example - description

turn: turn <factory> <colour> <pile> - Player turns, moving tiles from factory to the player pile line.
discard: discard <factory> <colour> - Player turns, moving tiles from factory to the player discard pile.
display: display - Displays all factories and the active players mosaic.
save: save <filename> - Saves the game to the specified filename.
help: help - Prints this help menu.
exit OR ^D (EOF command): exit - Exits the game.

=== Start Round ===

TURN FOR PLAYER: a
Factories:
0: F 
1: R U L Y 
2: U U L Y 
3: B L B R 
4: B B R L 
5: U U R U 

Mosaic for a
1:         . || . . . . . 
2:       . . || . . . . . 
3:     . . . || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
broken: . . . . . . . 
> Game successfully saved to 'saves/game_menu_options:save.azul'
> See ya :D
